src/hydra-queue-runner/hydra-queue-runner.cc

```diff
@@ -875,6 +891,16 @@ int main(int argc, char * * argv)
                     buildOne = *b;
                 else
                     throw Error("‘--build-one’ requires a build ID");
+            } else if (*arg == "--port") {
```

Comment:
This should also be configurable via the config file, but also override-able here (e.g. the flag should take precedence).